templates/system/html.tpl.php

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html <?php print $rdf_namespaces; ?>>
+
+  <head>
+    <?php print $head; ?>
+    <title><?php print $head_title; ?></title>
+
+    <!-- Basic Page Needs -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <!-- Mobile Specific Metas -->
+    <meta name="HandheldFriendly" content="True">
+    <meta name="MobileOptimized" content="320">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Favicons -->
+    <!-- 128x128 -->
+    <link rel="icon" type="image/png" href="<?php print $theme_path; ?>/assets/img/favicons/favicon.png" />
+    <!-- 192x192, as recommended for Android http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android -->
+    <link rel="icon" type="image/png" sizes="192x192" href="<?php print $theme_path; ?>/assets/img/favicons/favicon-192.png" />
```

Comment:
@AlexBrown-appnovation Great to see your work on this! Can you (create and) include the referenced image assets in this PR so the hrefs will work?